scripts/screenshot.js

```diff
@@ -26,9 +25,12 @@ async function run() {
   try {
     await page.setViewport({ width: 1200, height: 800 });
     await page.goto(uri, {
-      timeout: 30000,
+      timeout: 60000,
       waitUntil: 'networkidle2'
     });
+    // wait a little bit longer for the page to potentially finish rendering
+    // see: https://github.com/sul-dlss/was_robot_suite/issues/570
+    await sleep(10);
```

Comment:
Wait 10 seconds more once the page resources appear to have loaded. This should let any additional rendering happen.